client/src/includes/sidePanel.js

```diff
@@ -36,7 +36,7 @@ export default function initSidePanel() {
 
     const body = document.querySelector('body');
     const selectedPanel = document.querySelector(
```

Comment:
This variable was only used for the following code a few lines below:

```ts
    // Abort if panelName is specified but it does not exist
    if (panelName && !selectedPanel) return;
```

Instead of querying the toggle, query the actual panel itself so we can use the variable to dispatch the show event outside of the loop scope for dispatching the hide events.